react/features/base/settings/functions.js

```diff
@@ -81,3 +83,17 @@ export function getPropertyValue(
 
     return undefined;
 }
+
+/**
+ * Returns the settings object (or empty object). This is a helper function to
+ * avoid using the redux subtree name every time we need the settings in the
+ * app.
+ *
+ * @public
+ * @param {Object|Function} stateful - The redux state object or
+ * {@code getState} function.
+ * @returns {Object}
+ */
+export function getSettings(stateful: Object | Function): Object {
+    return toState(stateful)['features/base/settings'] || {};
```

Comment:
Are you the extra defensive type or could the settings state really be falsy?

Reply:
Yes, I'm the extra defensive type when writing JS :( Maybe I should stop doing this, but it just feels good.

If only it was Java... <3